<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimeraWindow</class>
 <widget class="QMainWindow" name="PrimeraWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnMenu">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>711</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>MENÃš</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlatoslacarta">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>711</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PLATOS A LA CARTA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSandwiches">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>711</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SANDWICHES, JUGOS Y BEBIDAS</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>UTEC.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMENUTEC">
    <property name="title">
     <string>MENUTEC </string>
    </property>
   </widget>
   <addaction name="menuMENUTEC"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
